build.sbt

```diff
@@ -25,6 +25,8 @@ val javaDslApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-
 val javaApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-java/api/"
 val javaAndroidApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-android/api/"
 
+useGpg := true
```

Comment:
The java implementation in sbt-pgp is broken, so GPG needs to be used. CI installs the key ahead of time in a way that sbt-pgp can accept it.

I will try moving it to `publishSettings`